usr/src/tools/scripts/git-pbchk.py

```diff
@@ -163,7 +164,9 @@ def git_file_list(parent, paths=None):
 
     ret = set()
     for fname in p:
-        if fname and not fname.isspace() and fname not in ret:
+        res = git("diff %s HEAD %s" % (parent, fname))
```

Comment:
There aren't any files with whitespace in the repo, and your suggested change doesn't work.  So I'm going to go ahead with the changes as proposed unless you have a better idea.  Here's the error message:

```
Traceback (most recent call last):
  File "/opt/onbld/bin/git-pbchk", line 386, in <module>
    main(os.path.basename(sys.argv[0]), sys.argv[1:])
  File "/opt/onbld/bin/git-pbchk", line 382, in main
    func(git_root(), parent_branch, args)
  File "/opt/onbld/bin/git-pbchk", line 355, in pbchk
    run_checks(root, parent, cmds)
  File "/opt/onbld/bin/git-pbchk", line 326, in run_checks
    output=s)
  File "/opt/onbld/bin/git-pbchk", line 251, in copyright
    for f in flist():
  File "/opt/onbld/bin/git-pbchk", line 208, in ret
    for f in git_file_list(parent, paths):
  File "/opt/onbld/bin/git-pbchk", line 169, in git_file_list
    res = git(["diff", parent, "HEAD", fname])
### End git-pbchk output ###
  File "/opt/onbld/bin/git-pbchk", line 81, in git
    raise GitError(p.stdout.read())
AttributeError: 'NoneType' object has no attribute 'read'
```